### ISSUE TITLE
Errors on react-native 0.33.0

### ISSUE THREAD

**Issue by cymen** [andyearnshaw/Intl.js#225]
This module was working with react-native 0.30.0 however it stopped working with 0.33.0 with this error:

```log
09-16 13:18:00.770 30570 30612 E ReactNativeJS: Requiring unknown module "./locale-data/complete.js".If you are sure the module is there, try restarting the packager or running "npm install".
```

I initially thought this was due to `.bablerc` not being in `.npmignore` which is apparently becoming a good practice (see https://github.com/facebook/react-native/issues/4062). I tried removing `.babelrc` but it didn't fix the problem.

I didn't get much further but wanted to report it in case react-native support is desired.

**Comment by Bhullnatik** [comment]
To offer some more informations, these packages were not working due to a buggy implementation of the `browser` field in `package.json` in React Native (See https://github.com/facebook/react-native/issues/9854).

It has since been fixed in the `0.35-rc` version, but now you must import specifically the locales you wish to use like so:

```js
import 'intl/locale-data/jsonp/en.js';
```

since it will not load all locales automatically now.

**Comment by caridy** [comment]
thanks for the clarificatino @Bhullnatik, closing this!
